corehq/apps/dashboard/templates/dashboard/base.html

```diff
@@ -25,6 +25,8 @@ <h1 class="mb-3 mt-5">
     </h1>
     {% if is_paused %}
       {% include 'hqwebapp/partials/bootstrap5/paused_plan_notice.html' %}
+    {% elif should_show_pending_notice %}
+      {% include 'hqwebapp/partials/bootstrap5/pending_plan_notice.html' %}
```

Comment:
Can you move `pending_plan_notice.html` up a directory? Only files that have both a B3 and B5 version should be in bootstrap3/bootstrap5 directories.